frontend/packages/ceph-storage-plugin/integration-tests/tests/add-capacity.scenario.ts

```diff
@@ -36,98 +37,119 @@ const expansionObjects = {
   updatedPods: { items: [] as PodKind[] },
 };
 
-describe('Check add capacity functionality for ocs service', () => {
-  beforeAll(async () => {
-    [expansionObjects.clusterJSON] = storageCluster.items;
-    const name = getName(expansionObjects.clusterJSON);
-    expansionObjects.previousCnt = _.get(
-      expansionObjects.clusterJSON,
-      'spec.storageDeviceSets[0].count',
-    );
-    const uid = _.get(expansionObjects.clusterJSON, 'metadata.uid').toString();
-    expansionObjects.previousPods = JSON.parse(
-      execSync(`kubectl get pods -n ${NS} -o json`).toString(),
-    );
-
-    await goToInstalledOperators();
-    await click(ocsOp);
-    await click(storageClusterView);
-
-    await clickKebabAction(uid, 'Add Capacity');
-    await verifyFields();
-    await click(confirmButton);
-
-    const statusCol = storageClusterRow(uid).$('td:nth-child(4)');
-
-    // need to wait as cluster states fluctuates for some time. Waiting for 2 secs for the same
-    await browser.sleep(2 * SECOND);
-
-    await browser.wait(until.textToBePresentInElement(statusCol, 'Progressing'));
-    await browser.wait(
-      until.textToBePresentInElement(statusCol.$('span.co-icon-and-text span'), 'Ready'),
-    );
-
-    expansionObjects.updatedClusterJSON = JSON.parse(
-      execSync(`kubectl get -o json -n ${NS} ${KIND} ${name}`).toString(),
-    );
-    expansionObjects.updatedCnt = _.get(
-      expansionObjects.updatedClusterJSON,
-      'spec.storageDeviceSets[0].count',
-    );
-    expansionObjects.updatedPods = JSON.parse(
-      execSync(`kubectl get pod -o json -n ${NS}`).toString(),
-    );
-  }, EXPAND_WAIT);
-
-  it('Newly added capacity should takes into effect at the storage level', () => {
-    // by default 2Tib capacity is being added
-    expect(expansionObjects.updatedCnt - expansionObjects.previousCnt).toEqual(1);
-  });
+describe('Check availability of cluster', () => {
+  if (cephValue.items[0]) {
```

Comment:
Its better to keep `cephValue.items[0]` in some variable and use it, as its used at several places.